.github/workflows/zero-downtime-tests.yml

```diff
@@ -1,13 +1,456 @@
-name: Zero-Downtime Tests
+name: ZeroDowntime Tests
 on:
-  workflow_dispatch:
+  workflow_dispatch: # run integration tests only when triggered manually
+    inputs:
+      branch:
+        description: 'Take CI build artifacts from branch (e.g., master, release-x.y.z)'
+        required: true
+        default: 'master'
+      upgradeTo:
+        description: 'Helm chart path to run upgrade command'
+        required: true
+        default: '0.15.0-dev-PR-7504'
```

Comment:
I wouldn't provide a default here. This version will always change and I don't think we can provide a sensible default value